hc/front/views.py

```diff
@@ -211,11 +211,12 @@ def channels(request):
                     check = Check.objects.get(code=code)
                 except Check.DoesNotExist:
                     return HttpResponseBadRequest()
-                if check.user != request.user:
+                if check.user_id != request.user.id:
                     return HttpResponseForbidden()
                 new_checks.append(check)
 
-        channel.checks = new_checks
```

Comment:
It's a m2m relation. I think it does:
https://docs.djangoproject.com/en/1.8/topics/db/examples/many_to_many/
"Relation sets can be assigned. Assignment clears any existing set members"